Backend/src/main/java/com/onbank/api/service/implementation/TransferServiceImpl.java

```diff
@@ -21,6 +24,10 @@
 
     @Override
     public Transfer createTransfer(Transfer transfer) {
+        Authentication auth = SecurityContextHolder.getContext().getAuthentication();
```

Comment:
Czy w przypadku każdej operacji będziemy zmuszeni do wyciągania kontekstu użytkownika w ten sposób? Jeżeli tak, to żeby uniknąć replikacji proponuje wynieść tę operację do odrębnej klasy z metodą statyczną (jeżeli w ten sposób to zadziała i kontekst się przechowa) bądź do serwisu